spec/tests/component.fmf

```diff
@@ -10,10 +10,16 @@ description:
     given component or package. As they do not always have to be
     stored in the same repository and because many tests cover
     multiple components a dedicated field is needed. Should be a
-    ``list of strings``.
+    ``string`` or a ``list of strings``. Component name usually
+    corresponds to the source package name.
 
 example: |
+    component: libselinux checkpolicy
```

Comment:
This is not a standard notion, if we enable it like this here, we should document it explicitly here or somewhere else.

Reply:
Yeah, let's discuss this today on the hacking session. Either we remove support for splitting on white-space/comma or we clearly document it.

Reply:
yeah, definitely

Reply:
Split on white-space/comma removed.